Bakker/models/bakker_verkoop.py

```diff
@@ -0,0 +1,328 @@
+                'active_ids': self.ids,
+                'active_model': 'bakker_verkoop',
+                'report_action': self.env.ref('Bakker.report_bakker_factuur').id,
+            }
+        }
+    
+    def action_configure_print(self):
+        """Open document layout configuratie voor factuur"""
+        return {
+            'type': 'ir.actions.act_window',
+            'name': 'Configure Document Layout',
+            'res_model': 'base.document.layout',
+            'view_mode': 'form',
+            'target': 'new',
+            'context': {
+                'default_report_layout': 'web.external_layout_standard',
+                'default_logo': True,
+                'default_company_details': True,
+                'active_ids': self.ids,
+                'active_model': 'bakker_verkoop',
+                'report_action': self.env.ref('Bakker.report_bakker_factuur').id,
+            }
+        }
+    
+    def action_print_with_layout(self):
+        """Print factuur met document layout opties"""
+        return {
+            'type': 'ir.actions.act_window',
+            'name': 'Print Factuur',
+            'res_model': 'base.document.layout',
+            'view_mode': 'form',
+            'target': 'new',
+            'context': {
+                'default_report_layout': 'web.external_layout_standard',
+                'default_logo': True,
+                'default_company_details': True,
+                'active_ids': self.ids,
+                'active_model': 'bakker_verkoop',
+                'report_action': self.env.ref('Bakker.report_bakker_factuur').report_action(self),
+                'close_on_report_download': True,
+            }
+        }
+        
+        
+class BakkerVerkoopWizard(models.TransientModel):
+    _name = 'bakker.verkoop.wizard'
+    _description = 'Wizard voor het verkopen van koeken'
+    
+    koek_id = fields.Many2one('bakker_koeken', string='Koek', required=True)
+    partner_id = fields.Many2one('res.partner', string='Klant', required=True)
+    aantal = fields.Integer(string='Aantal', required=True, default=1)
+    prijs_per_stuk = fields.Float(string='Prijs per stuk', required=True)
+    korting_percentage = fields.Float(string='Korting (%)', default=0.0)
+    finale_prijs = fields.Float(string='Finale Prijs per stuk', compute='_compute_finale_prijs')
+    totaal_bedrag = fields.Float(string='Totaal Bedrag', compute='_compute_totaal_bedrag')
+    beschikbare_voorraad = fields.Integer(related='koek_id.voorraad_koek', string='Beschikbare Voorraad')
+    direct_betaald = fields.Boolean(string='Direct Betaald', default=True)
+    betaal_methode = fields.Selection([
+        ('cash', 'Contant'),
+        ('card', 'Bankkaart'),
+        ('digital', 'Digitaal')
+    ], string="Betaal Methode", default='cash')
+    
+    # Gerelateerde velden van de klant
+    klant_email = fields.Char(string="Email", related='partner_id.email', readonly=True)
```

Comment:
```suggestion
    klant_email = fields.Char(string="Email", related='partner_id.email')
```
A related field should be readonly=True by default if I remember well